src/commands/RemoteControl.ts

```diff
@@ -0,0 +1,104 @@
+import * as readline from 'readline';
+import type { RokuKey } from '../index';
+import { rokuDeploy, util } from '../index';
+
+export class RemoteControl {
```

Comment:
Let's call this class `RemoteControlCommand` and the file `RemoteControlCommand.ts` to align with the other files in this folder.